go.mod

```diff
@@ -6,3 +6,8 @@ require (
 	github.com/golang-jwt/jwt/v4 v4.4.1
 	github.com/opencontainers/go-digest v1.0.0
 )
+
+require (
+	github.com/go-ldap/ldap v3.0.3+incompatible // indirect
```

Comment:
How is that `ldap` is marked as indirect when it is directly referenced in our code?

Reply:
I should have run `go mod tidy`. Also, I realized that I should use go-ldap v3 as mentioned [here](https://github.com/go-ldap/ldap#go-modules). Thanks for pointing this out. Will raise a new revision for this PR.